src/Mod/TechDraw/App/AppTechDrawPy.cpp

```diff
@@ -1,7 +1,7 @@
 /***************************************************************************
- *   Copyright (c) Jürgen Riegel          (juergen.riegel@web.de) 2002     *
- *   Copyright (c) WandererFan            (wandererfan@gmail.com) 2016     *
-*                                                                         *
+ *   Copyright (c) 2002 Jürgen Riegel <juergen.riegel@web.de>              *
+ *   Copyright (c) 2016 WandererFan <wandererfan@gmail.com>                *
+ *                                                                         *

```

Comment:
Personally, I don't think you need to repeat the word Copyright for each author. It's obvious after the first one that all of them have that copyright. What's important is the year, to know when an author did a significant change.

Reply:
I don't know.

Reply:
I think we should just move forward with this PR as is. We can discuss the cosmetic change later.